tests/features/collection/og.feature

```diff
@@ -79,3 +80,27 @@ Feature: Organic Groups integration
     Then I should see the success message "You are no longer a member of Folk Dance and Song Society."
     And I should see the "Join this collection" button
     And the "Folk Dance and Song Society" collection should have 1 member
+
+  @javascript
+  Scenario: Edit a Collection
+    Given collections:
+      | uri                              | logo     | name                         | abstract                                    | access url                              | closed  | creation date     | description                                                                                                         | elibrary creation  | moderation  | modification date  | owner |
+      | http://joinup.eu/collection/dog  | logo.png | Überwaldean Land Eels        | Read up on all about <strong>dogs</strong>  | http://dogtime.com/dog-breeds/profiles  | 1       | 28-01-1995 12:05  | The Afghan Hound is elegance personified.                                                                           | 0                  | 1           |                    |       |
+      | http://joinup.eu/collection/cat  | logo.png | Folk Dance and Song Society  | Cats are cool!                              | http://mashable.com/category/cats/      | 0       | 28-01-1995 12:06  | The domestic cat (Felis catus or Felis silvestris catus) is a small usually furry domesticated carnivorous mammal.  | 1                  | 0           |                    |       |
+    And users:
+      | name             | roles         |
+      | Collection admin | administrator |
+      | Madame Sharn     |               |
+      | Goodie Whemper   |               |
+    # Administrators can edit the collection.
+    # @fixme Run this as facilitator or collection owner.
+    Given I am logged in as "Collection admin"
```

Comment:
Add a link here to ISAICP-2190. We will add the roles in that issue, so this can be fixed very soon.